apps/andi/lib/andi_web/live/dataset_live_view.ex

```diff
@@ -3,6 +3,8 @@ defmodule AndiWeb.DatasetLiveView do
   alias AndiWeb.Router.Helpers, as: Routes
   alias AndiWeb.DatasetLiveView.Table
 
+  @ingested_time_topic "ingested_time_topic"
```

Comment:
This is hardcoded in two places. Should it be set in configuration or maybe as a function on andi.ex?

Reply:
If we wanted it to be configurable then I'd agree, but at this point I don't see the value/jsr.